opencog/openpsi/openpsi.scm

```diff
@@ -7,13 +7,23 @@
 
 (load-from-path "opencog/openpsi/action-selector.scm")
 (load-from-path "opencog/openpsi/demand.scm")
+(load-from-path "opencog/openpsi/control.scm")
 (load-from-path "opencog/openpsi/rule.scm")
 (load-from-path "opencog/openpsi/main.scm")
-(load-from-path "opencog/openpsi/modulator.scm")
-(load-from-path "opencog/openpsi/sec.scm")
 (load-from-path "opencog/openpsi/utilities.scm")
-(load-from-path "opencog/openpsi/entity-defs")
-(load-from-path "opencog/openpsi/interaction-rules.scm")
-(load-from-path "opencog/openpsi/updater.scm")
+
+(define-public (load-openpsi-in-development)
+; This is a temporary hack to get past installing openpsi to get access to
```

Comment:
@eddiemonroe I added this utility function to help in debugging the your code from the one in usage by rbs & HEAD. Just FYI